RecoLocalTracker/SiPixelClusterizer/plugins/gpuCalibPixel.h

```diff
@@ -18,7 +19,12 @@ namespace gpuCalibPixel {
   constexpr float VCaltoElectronOffset = -60;      // L2-4: -60 +- 130
   constexpr float VCaltoElectronOffset_L1 = -670;  // L1:   -670 +- 220
 
-  template <bool isRun2>
+  constexpr float ElectronPerADCGain = 600;
+  constexpr int8_t Phase2ReadoutMode = 3;
+  constexpr uint16_t Phase2DigiBaseline = 1500;
+  constexpr uint8_t Phase2KinkADC = 8;
+
+  template <bool isRun2, bool isUpgrade>
```

Comment:
How about using an enumeration instead with a single template parameter? On the other hand, the difference with `isUpgrade` looks so substantial that a separate function could be considered.

Reply:
Indeed, as above didn't know if it was better/more readable to split them or to "duplicate". Probably yes, it's easier to have an independent one.